src/Twilio/Types/SubscribeRule.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Twilio.Converters;
+
+/// <summary>
+/// Subscribe Rule - A single Subscribe Rule for a Participant
```

Comment:
Link to docs and rule validation method would be helpful here (since you've set the bar already in java).